pkg/gce-pd-csi-driver/controller.go

```diff
@@ -159,7 +159,7 @@ const (
 )
 
 var (
-	validResourceApiVersions = map[string]bool{"v1": true, "alpha": true, "beta": true}
+	validResourceApiVersions = map[string]bool{"v1": true, "alpha": true, "beta": true, "staging_v1": true, "staging_beta": true, "staging_alpha": true}
```

Comment:
@mattcary Is it okay we put this on a public repo? It still requires permission to access the staging env.

Reply:
We also have this in gce.go -> computeVersionMap. I can instead define a variable or something similar to be used at both places instead of hardcoded values. 
Note: They will still be hardcoded at variable initialization

Reply:
I have removed the other map in favor of building path programatically so we will need to keep this map as is